bokehjs/src/coffee/core/ui_events.coffee

```diff
@@ -238,24 +252,22 @@ export class UIEvents
     @_trigger('rotate:end', e)
 
   _mouse_enter: (e) ->
-    @_bokify_jq(e)
-    @_trigger('move:enter', e)
+    @_bokify_point_event(e)
+    @trigger('move:enter', e)
```

Comment:
@jlstevens, OK. It used to be actually `@trigger()`. I changed it to `@_trigger()` in PR #5349, to gain full control over event flow. Now seeing this change to original, I was wondering what will break when I change it to what I need.